### PR TITLE
REL: v2.0.0b2 version bump

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -43,6 +43,7 @@ requirements:
     - pyqt  # [not osx]
     - tk 8.5*  # [linux]
     - functools32  # [py2k]
+    - subprocess32  # [py2k and unix]
```
Comment (pelson) [inline recipe/meta.yaml]: Unfortunately this is missing. 😢

**Comment by pelson** [comment]
Congratulations @tacaswell! 🎉

File: recipe/meta.yaml (hunk 1 above)
Comment (tacaswell) [inline recipe/meta.yaml]: Did I mark the selectors up correctly?  I started to take a run at packaging this last night, but gave up and went to sleep.

Comment (tacaswell) [inline recipe/meta.yaml]: ok, got it working this morning, PR coming

Comment (tacaswell) [inline recipe/meta.yaml]: https://github.com/conda-forge/staged-recipes/pull/1027

**Comment by pelson** [comment]
Recommendation: make ICU a *temporary* dependency on Linux until the problem is resolved.

**Comment by tacaswell** [comment]
So, the sha256 for the release has changed.....

**Comment by jjhelmus** [comment]
> So, the sha256 for the release has changed.....

Had a chat with @ocefpaf about this, he mentioned that he has seen this before in #38 and in another package.  Will need to dig into this further if we see this occurring often.

**Comment by tacaswell** [comment]
OK, I have sorted this out.

The issue seems to be that versioneer has a some (very useful) hooks that run when you create the tarball it injects information about the current git tree into the `_version.py` file (so we get automagical versions!).  The issue is that it seems to be injecting the names of all of the git references which point at the current hash.   Below is a diff of the two tarballs I downloaded ('mpl_original` from when I tagged, `mpl_new` from today).  In between we merged something to 2.x, the branch moved, and hence the tar ball changed (which means we have determined that gh re-generates the tarballs on demand). 

```diff
17:34 $ diff -ur mpl_orginal mpl_new
diff -ur mpl_orginal/lib/matplotlib/_version.py mpl_new/lib/matplotlib/_version.py
--- mpl_orginal/lib/matplotlib/_version.py      2016-07-13 01:14:37.000000000 -0500
+++ mpl_new/lib/matplotlib/_version.py  2016-07-13 01:14:37.000000000 -0500
@@ -20,7 +20,7 @@                                                                                                                                                                                                                                                                         
     # setup.py/versioneer.py will grep for the variable names, so they must                                                                                                                                                                                                              
     # each be defined on a line of their own. _version.py will just call                                                                                                                                                                                                                 
     # get_keywords().                                                                                                                                                                                                                                                                    
-    git_refnames = " (tag: v2.0.0b2, v2.x)"                                                                                                                                                                                                                                              
+    git_refnames = " (tag: v2.0.0b2)"                                                                                                                                                                                                                                                    
     git_full = "be2c5fd53598a83a1b0285f6ad5c54e826133745"                                                                                                                                                                                                                                
     keywords = {"refnames": git_refnames, "full": git_full}                                                                                                                                                                                                                              
     return keywords
```

This probably means that many of the existing recipes will no longer build.

**Pull request merged**